addon/components/light-table/cells/link.js

```diff
@@ -1,11 +1,12 @@
 import { isEmpty } from '@ember/utils';
 import BaseCell from 'ember-light-table/components/cells/base';
+import layout from 'ember-light-table-cell-type-link/templates/components/light-table/cells/link'
 
 /**
  * @class LinkCell
  * @extends LightTable.BaseCell
  */
-export default class extends BaseCell {
+export default class extends BaseCell.extend({ layout }) {
```

Comment:
There are some magic properties like `layout` (I believe) that do not work with the new ES6 class syntax yet.